resources/github-comment-markdown.template

```diff
@@ -79,7 +79,7 @@ ${errorStackTrace}
 <% stepsErrors = stepsErrors?.findAll{it?.result == "FAILURE" &&
                                       !it?.displayName?.contains('Notifies GitHub') &&
                                       !it?.displayName?.contains('Archive JUnit') &&
-                                      !it?.displayDescription?.contains('githubPrCheckApproved') &&
+                                      errorGithubPrCheckApproved &&
```

Comment:
Using the variable from https://github.com/elastic/apm-pipeline-library/pull/1714/files#diff-9459565d9aa2eb20956e84e6e5c2e6b3ae1c3902aa3f41b63c4e5b26ed52ca34R77